samples/python2/_tutorials_url.txt

```diff
@@ -0,0 +1,2 @@
+OpenCV-Python Tutorials
+http://opencv-python-tutroals.readthedocs.org/en/latest/py_tutorials/py_tutorials.html
```

Comment:
Oh, this link is indeed outdated, as the author wrote here: https://github.com/abidrahmank/OpenCV2-Python-Tutorials/issues/39#issuecomment-64357484

I commit adapted urls as soon as possible.